doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -38,9 +38,11 @@ They return a structure that contains the different file names::
     >>> # The different files
     >>> print(list(haxby_files.keys()))
     ['mask_house_little', 'anat', 'mask_house', 'mask_face', 'func', 'session_target', 'mask_vt', 'mask_face_little']
-    >>> #  Path to first functional file
+    >>> # Path to first functional file
     >>> print(haxby_files.func[0] # doctest: +ELLIPSIS)
```

Comment:
Hmm while you are at it, shouldn't the parenthesis be before the comment, i.e.:

```
>>> print(haxby_files.func[0])  # doctest: +ELLIPSIS
```

rather than

```
>>> print(haxby_files.func[0]  # doctest: +ELLIPSIS)
```

probably comes from a global search and replace for the Python 3 compatibility ...